[2024-09-05T00:34:26.888+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-05T00:34:27.021+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-09-05T00:34:27.058+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-09-05T00:34:27.242+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-05T00:34:27.495+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): translate_and_clean_data> on 2024-08-01 00:00:00+00:00
[2024-09-05T00:34:27.506+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_dag', 'translate_and_clean_data', 'scheduled__2024-08-01T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmpwsb6sug7']
[2024-09-05T00:34:27.511+0000] {standard_task_runner.py:105} INFO - Job 39: Subtask translate_and_clean_data
[2024-09-05T00:34:27.503+0000] {warnings.py:112} WARNING - /usr/local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=290) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-05T00:34:27.514+0000] {standard_task_runner.py:72} INFO - Started process 294 to run task
[2024-09-05T00:34:27.757+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-01T00:00:00+00:00 [running]> on host b778581f4b0f
[2024-09-05T00:34:29.419+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='translate_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-01T00:00:00+00:00'
[2024-09-05T00:34:29.429+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-05T00:34:29.543+0000] {logging_mixin.py:190} INFO - Recuperando DataFrame de pronósticos desde XCom
[2024-09-05T00:34:29.775+0000] {logging_mixin.py:190} INFO - Número de registros en pronósticos: 18
[2024-09-05T00:34:29.849+0000] {logging_mixin.py:190} INFO - Pronósticos antes de limpiar y traducir:    cve_mun            municipio  ... wind_kph  humidity
0        1    AMEALCO DE BONFIL  ...     16.9        77
1        2      PINAL DE AMOLES  ...     19.8        77
2        3          ARROYO SECO  ...     19.1        82
3        4  CADEREYTA DE MONTES  ...     16.2        78
4        5                COLON  ...     19.8        77

[5 rows x 10 columns]
[2024-09-05T00:35:33.414+0000] {logging_mixin.py:190} INFO - Pronósticos después de limpiar y traducir:    cve_mun            municipio  ... viento_kph  humedad
0        1    Amealco De Bonfil  ...       16.9       77
1        2      Pinal De Amoles  ...       19.8       77
2        3          Arroyo Seco  ...       19.1       82
3        4  Cadereyta De Montes  ...       16.2       78
4        5                Colon  ...       19.8       77

[5 rows x 10 columns]
[2024-09-05T00:35:33.540+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-05T00:35:33.587+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-05T00:35:33.613+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=translate_and_clean_data, run_id=scheduled__2024-08-01T00:00:00+00:00, execution_date=20240801T000000, start_date=20240905T003427, end_date=20240905T003533
[2024-09-05T00:35:33.768+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-05T00:35:33.850+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-05T00:35:33.891+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
[2024-09-05T18:06:52.486+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-05T18:06:52.555+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-09-05T18:06:52.572+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-09-05T18:06:52.573+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-05T18:06:52.650+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): translate_and_clean_data> on 2024-08-01 00:00:00+00:00
[2024-09-05T18:06:52.655+0000] {warnings.py:112} WARNING - /usr/local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=434) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-05T18:06:52.657+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_dag', 'translate_and_clean_data', 'scheduled__2024-08-01T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmpd2xkqjrm']
[2024-09-05T18:06:52.659+0000] {standard_task_runner.py:72} INFO - Started process 439 to run task
[2024-09-05T18:06:52.659+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask translate_and_clean_data
[2024-09-05T18:06:52.768+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-01T00:00:00+00:00 [running]> on host 0b8eaea5e88c
[2024-09-05T18:06:53.149+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='translate_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-01T00:00:00+00:00'
[2024-09-05T18:06:53.154+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-05T18:06:53.181+0000] {logging_mixin.py:190} INFO - Recuperando DataFrame de pronósticos desde XCom
[2024-09-05T18:06:53.289+0000] {logging_mixin.py:190} INFO - Número de registros en pronósticos: 18
[2024-09-05T18:06:53.310+0000] {logging_mixin.py:190} INFO - Pronósticos antes de limpiar y traducir:    cve_mun            municipio  ... wind_kph  humidity
0        1    AMEALCO DE BONFIL  ...     18.0        78
1        2      PINAL DE AMOLES  ...     16.6        75
2        3          ARROYO SECO  ...     17.3        84
3        4  CADEREYTA DE MONTES  ...     14.0        79
4        5                COLON  ...     16.6        75

[5 rows x 10 columns]
[2024-09-05T18:07:54.666+0000] {logging_mixin.py:190} INFO - Pronósticos después de limpiar y traducir:    cve_mun            municipio  ... viento_kph  humedad
0        1    Amealco De Bonfil  ...       18.0       78
1        2      Pinal De Amoles  ...       16.6       75
2        3          Arroyo Seco  ...       17.3       84
3        4  Cadereyta De Montes  ...       14.0       79
4        5                Colon  ...       16.6       75

[5 rows x 10 columns]
[2024-09-05T18:07:56.296+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-05T18:07:56.374+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-05T18:07:56.401+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=translate_and_clean_data, run_id=scheduled__2024-08-01T00:00:00+00:00, execution_date=20240801T000000, start_date=20240905T180652, end_date=20240905T180756
[2024-09-05T18:07:56.528+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-05T18:07:56.566+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-05T18:07:56.621+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
